TACC/Core-CMS
# feat/tup-630: Migrate django.cms.blog.app.item to core-cms

## Overview

Migrated django.cms.blog.app.item to core-cms

## Related

- [TUP-630](https://jira.tacc.utexas.edu/browse/TUP-630)
- Migrate from [django.cms.blog.app.item](https://github.com/TACC/tup-ui/blob/main/apps/tup-cms/src/taccsite_custom/tup_cms/static/tup_cms/css/for-core-cms/components/django.cms.blog.app.item.css)


## Changes

CSS includes bold and colored lettering of article date.

## Testing

1. Make sure that posts have colors, and are weighted
2. Make sure that css is coming from app.blog.css

## UI

| Before | After |
| ------ | ------ |
| <img width="873" alt="Screenshot 2023-10-18 at 3 15 12 PM" src="https://github.com/TACC/Core-CMS/assets/63771558/0300ee72-c3ee-4472-a0e5-c6724dafc600">  | <img width="1451" alt="Screenshot 2023-10-18 at 4 27 19 PM" src="https://github.com/TACC/Core-CMS/assets/63771558/dd0f070c-e79e-4ada-b217-5d11374ff992"> |
